cfgov/unprocessed/js/organisms/FormSubmit.js

```diff
@@ -5,11 +5,16 @@ import {
   BaseTransition,
   EventObserver,
 } from '@cfpb/cfpb-atomic-component';
-import ERROR_MESSAGES from '../config/error-messages-config.js';
 import Notification from '../molecules/Notification.js';
 import { scrollIntoView } from '../modules/util/scroll.js';
 
-const FORM_MESSAGES = ERROR_MESSAGES.FORM.SUBMISSION;
+const FORM_MESSAGES = {
```

Comment:
Just to clarify: these aren't used any longer with FLs, right, after this PR? The error messages should all come from the backend, [for example](https://www.consumerfinance.gov/about-us/blog/?title=&topics=ans&language=ans&from_date=&to_date=).

Reply:
FormSubmit is used in the email signup and popup (nudgy guy), which still does validation on the client-side.

While looking at this, I added a minor commit to standardize the validator imports https://github.com/cfpb/consumerfinance.gov/pull/7770/commits/be94a9e70ee122d4d1a3b7fa7d9b4cac1d37e195